src/main/java/org/bytecodeparser/utility/ThrowingFunction.java

```diff
@@ -1,20 +1,16 @@
 package org.bytecodeparser.utility;
 
-import org.bytecodeparser.instruction.InstructionArguments;
-
-import java.io.DataInputStream;
-import java.io.IOException;
 import java.util.function.Function;
 
 @FunctionalInterface
 public interface ThrowingFunction<T, E extends Exception, V> {
     V apply(T t) throws E;
 
-    static Function<DataInputStream, InstructionArguments> resolveStrategyWrapper(ThrowingFunction<DataInputStream, IOException, InstructionArguments> function) {
-        return dataInputStream -> {
+    static <T, E extends Exception, V> Function<T, V> throwableWrapper(ThrowingFunction<T, E, V> function) {
```

Comment:
Item 68 eff java, convention for type param - X is type parameters for exceptions

Reply:
V -> R

Reply:
done